Description: Update .desktop files to match current specification
Author: Olly Betts <olly@survex.com>
Forwarded: http://trac.wxwidgets.org/ticket/14345
Applied-Upstream: http://trac.wxwidgets.org/changeset/71590 (2.9.4)
Last-Update: 2012-05-28

Index: wx/wxPython/distrib/PyAlaMode.desktop
===================================================================
--- wx.orig/wxPython/distrib/PyAlaMode.desktop	2012-05-29 05:50:18.000000000 +1200
+++ wx/wxPython/distrib/PyAlaMode.desktop	2012-05-29 05:50:18.000000000 +1200
@@ -2,7 +2,7 @@
 Name=PyAlaMode
 Comment=GUI Python Shell with Filling and editor windows
 Exec=pyalamode
-Icon=PyCrust.png
+Icon=PyCrust
 Terminal=false
 Type=Application
-Categories=Application;Development;
+Categories=Development;
Index: wx/wxPython/distrib/Editra.desktop
===================================================================
--- wx.orig/wxPython/distrib/Editra.desktop	2012-05-29 05:50:18.000000000 +1200
+++ wx/wxPython/distrib/Editra.desktop	2012-05-29 05:50:18.000000000 +1200
@@ -2,7 +2,7 @@
 Name=Editra
 Comment="Programmer's Text Editor"
 Exec=editra
-Icon=editra.png
+Icon=editra
 Terminal=false
 Type=Application
-Categories=Application;Development;
+Categories=Development;
Index: wx/wxPython/distrib/PyShell.desktop
===================================================================
--- wx.orig/wxPython/distrib/PyShell.desktop	2012-05-29 05:50:18.000000000 +1200
+++ wx/wxPython/distrib/PyShell.desktop	2012-05-29 05:50:18.000000000 +1200
@@ -2,7 +2,7 @@
 Name=PyShell
 Comment=GUI Python Shell
 Exec=pyshell
-Icon=PyCrust.png
+Icon=PyCrust
 Terminal=false
 Type=Application
-Categories=Application;Development;
+Categories=Development;
Index: wx/wxPython/distrib/PySlices.desktop
===================================================================
--- wx.orig/wxPython/distrib/PySlices.desktop	2012-05-29 05:50:18.000000000 +1200
+++ wx/wxPython/distrib/PySlices.desktop	2012-05-29 05:50:18.000000000 +1200
@@ -2,7 +2,7 @@
 Name=PySlices
 Comment=GUI Python Shell with Filling
 Exec=pyslices
-Icon=PySlices.png
+Icon=PySlices
 Terminal=false
 Type=Application
-Categories=Application;Development;
+Categories=Development;
Index: wx/wxPython/distrib/XRCed.desktop
===================================================================
--- wx.orig/wxPython/distrib/XRCed.desktop	2012-05-29 05:50:18.000000000 +1200
+++ wx/wxPython/distrib/XRCed.desktop	2012-05-29 05:50:18.000000000 +1200
@@ -2,7 +2,7 @@
 Name=XRCed
 Comment=wxPython XRC resource editor
 Exec=xrced
-Icon=XRCed.png
+Icon=XRCed
 Terminal=false
 Type=Application
-Categories=Application;Development;
+Categories=Development;
Index: wx/wxPython/distrib/PyCrust.desktop
===================================================================
--- wx.orig/wxPython/distrib/PyCrust.desktop	2012-05-29 05:50:18.000000000 +1200
+++ wx/wxPython/distrib/PyCrust.desktop	2012-05-29 05:50:18.000000000 +1200
@@ -2,7 +2,7 @@
 Name=PyCrust
 Comment=GUI Python Shell with Filling
 Exec=pycrust
-Icon=PyCrust.png
+Icon=PyCrust
 Terminal=false
 Type=Application
-Categories=Application;Development;
+Categories=Development;
Index: wx/wxPython/distrib/PySlicesShell.desktop
===================================================================
--- wx.orig/wxPython/distrib/PySlicesShell.desktop	2012-05-29 05:50:18.000000000 +1200
+++ wx/wxPython/distrib/PySlicesShell.desktop	2012-05-29 05:50:18.000000000 +1200
@@ -3,7 +3,7 @@
 Name=PySlicesShell
 Comment=GUI Python Shell
 Exec=pysliceshell
-Icon=PySlices.png
+Icon=PySlices
 Terminal=false
 Type=Application
-Categories=Application;Development;
+Categories=Development;
